rules/windows/file_change/file_change_win_unusual_modification_by_dns_exe.yml

```diff
@@ -0,0 +1,23 @@
+title: Unusual File Modification by dns.exe
+id: 9f383dc0-fdeb-4d56-acbc-9f9f4f8f20f3
+status: experimental
+author: Tim Rauch
+date: 2022/09/27
+description: Detects an unexpected file being modified by dns.exe which my indicate activity related to remote code execution or other forms of exploitation as seen in CVE-2020-1350 (SigRed)
+references:
+    - https://www.elastic.co/guide/en/security/current/unusual-file-modification-by-dns.exe.html
+tags:
+    - attack.initial_access
+    - attack.t1133
+logsource:
+    category: file_change
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\dns.exe'
+    selection_not:
```

Comment:
Using `filter`  instead of "selection_not" is a best practice for the comprehension